lib/WorkflowRnaseq.groovy

```diff
@@ -184,9 +150,7 @@ class Workflow {
         return [ strandedness, sense, antisense, undetermined ]
     }
 
-    /*
-     * Get workflow summary for MultiQC
-     */
+    // Get workflow summary for MultiQC
     public static String paramsSummaryMultiqc(workflow, summary) {
```

Comment:
Maybe there is something specific to the `rnaseq` pipeline but otherwise, could this function not be moved to `workflowMain.groovy`

Reply:
Yup, its specific to the rnaseq workflow in this pipeline which is why I put it here.